core/src/trezor/strings.py

```diff
@@ -21,12 +21,6 @@ def format_amount(amount: int, decimals: int) -> str:
     return s
 
 
-def format_ordinal(number: int) -> str:
```

Comment:
please keep this and hide behind `if False`

Reply:
Done in [ccdccd9](https://github.com/trezor/trezor-firmware/pull/2525/commits/ccdccd9eee7c79ac232a344004703483d2717db6)